todo_app/data/db_items.py

```diff
@@ -0,0 +1,21 @@
+import os
+from todo_app.models.Item import Item
+import pymongo
+
+def get_collection():
+  client = pymongo.MongoClient(f'{os.getenv("DB_CONNECTION_STRING")}')
+  db = client[f'{os.getenv("DB_NAME")}']
+  return db.card
+
+def get_cards():
+  collection = get_collection()
+  cards = collection.find()
+  return [Item.from_db_card(card) for card in cards]
+
+def add_card(card_name):
+  collection = get_collection()
+  collection.insert_one({"name": card_name, "status": "To Do"})
+
+def change_card_status(card_id, status):
+  collection = get_collection()
+  collection.update_one({"_id": card_id}, {"$set": {"status": status}})
```

Comment:
Minor, but it's a good habit to convert the string `card_id` here to an `ObjectId` (like below) since some versions of MongoDB will refuse to do the type conversion for you, although CosmosDB does
```
from bson import ObjectId

...

collection.update_one({"_id": ObjectId(card_id)}, {"$set": {"status": status}})
```